### PR TITLE
PLAT-1010 | Import LucidChart using any string containing chart id

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-1010
Follow up to https://github.com/bethinkpl/wnl-platform/pull/1472

### CODE REVIEW

File: tests/Api/LucidCharts/LucidChartsTest.php [hunk 1]
```diff
@@ -2,29 +2,20 @@
 
 namespace Tests\Api\LessonProduct;
```
Comment (bognix) [inline tests/Api/LucidCharts/LucidChartsTest.php]: please update namespace, I noticed I copied this and didn't update.

File: lib/LucidChart/LucidChart.php [hunk 2]
```diff
@@ -2,25 +2,55 @@
...
 class LucidChart
 {
 	const LUCID_URL_PATTERN = 'https://www.lucidchart.com/documents/thumb/%s/0/0/NULL/%d';
 	const LUCID_IMAGE_SIZE = 2000;
+	const LUCID_BROKEN_IMAGE_PLACEHOLDER_SIZE = 170;
 
 	public $assetPath;
 	public $chartId;
 
-	public function createFromChartId($chartId) {
```
Comment (bognix) [inline lib/LucidChart/LucidChart.php]: this method was used in `Parser.php` and `MegaUltraSuperDuperChartUpdateScript.php`

Comment (rogatty) [reply lib/LucidChart/LucidChart.php]: Oops, that's what happens when the IDE can't find the class. Thanks, I'll fix it.

Comment (bognix) [reply lib/LucidChart/LucidChart.php]: Thankfuly Larastan and unit tests caught it as well

File: app/Contracts/CourseProviderFacade.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class CourseProvider extends Facade
+{
+	/**
+	 * Get the registered name of the component.
+	 *
+	 * @return string
+	 */
+	protected static function getFacadeAccessor()
+	{
+		return 'App\Contracts\CourseProvider';
```
Comment (kvas-damian) [inline app/Contracts/CourseProviderFacade.php]: `App\Contracts\CourseProvider::class`

File: lib/Bethink/BethinkFacade.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+class Bethink extends Facade
+{
+	/**
+	 * Get the registered name of the component.
+	 *
+	 * @return string
+	 */
+	protected static function getFacadeAccessor()
+	{
+		return 'Lib\Bethink\Bethink';
```
Comment (kvas-damian) [inline lib/Bethink/BethinkFacade.php]: `Lib\Bethink\Bethink::class`

File: lib/Invoice/InvoiceFacade.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+class Invoice extends Facade
+{
+	/**
+	 * Get the registered name of the component.
+	 *
+	 * @return string
+	 */
+	protected static function getFacadeAccessor()
+	{
+		return 'Lib\Invoice\Invoice';
```
Comment (kvas-damian) [inline lib/Invoice/InvoiceFacade.php]: avoid using strings

File: tests/Api/LucidCharts/LucidChartsTest.php [hunk 6]
```diff
@@ -1,47 +1,113 @@
...
+			->actingAs($user)
+			->json('POST', $this->url("/lucid_charts"), [
+				'chart_url' => $testChartUrl
+			])
+			->assertStatus(422);
+
+		$lucidChartGetChartIdFromStringExpectation->verify();
+	}
+
+	public function testLucidChartBrokenImage()
```
Comment (kvas-damian) [inline tests/Api/LucidCharts/LucidChartsTest.php]: Can we add a test for id only?

Comment (rogatty) [reply tests/Api/LucidCharts/LucidChartsTest.php]: Sure, but in the unit tests: https://github.com/bethinkpl/wnl-platform/pull/1479/commits/2a98a54da3da80b5ce4a54fb67ce67c22aea416b

**Pull request merged**